app/src/main/java/openfoodfacts/github/scrachx/openfood/AppFlavors.kt

```diff
@@ -21,5 +21,5 @@ object AppFlavors {
     const val OPF = "opf"
     const val OBF = "obf"
     @JvmStatic
-    fun isFlavors(vararg flavors: String) = flavors.contains(BuildConfig.FLAVOR)
+    fun isFlavors(vararg flavors: String) = flavors.contains(BuildConfig.FLAVOR_versionCode)
```

Comment:
I don't think this will work. Don't we have to update the strings? Or maybe use startsWith.

Reply:
I tested, this works.

Reply:
Because we have added a new dimension.